qubes/storage/reflink.py

```diff
@@ -227,14 +227,23 @@ def stop(self):
         else:
             if not self.snap_on_start:
                 self._size = self.size  # preserve manual resize of image
-            _remove_file(self._path_dirty)
-            _remove_file(self._path_clean)
+            try:
+                _remove_file(self._path_dirty)
+            except FileNotFoundError:
+                pass
+            try:
+                _remove_file(self._path_clean)
+            except FileNotFoundError:
+                pass
```

Comment:
This isn't necessary,  the reflink module's `_remove_file()` already suppresses `FileNotFoundError`.

Reply:
Indeed it does.